MAINTAINERS.yml

```diff
@@ -1095,7 +1173,93 @@ SPARC arch:
     labels:
         - "area: SPARC"
 
-STM32:
+Synopsys Platforms:
+    status: maintained
+    maintainers:
+        - ruudw
+    collaborators:
+        - abrodkin
+        - evgeniy-paltsev
+        - IRISZZW
+    files:
+        - soc/arc/
+        - boards/arc/
+    labels:
+        - "platform: Synopsys"
+
+Intel Platforms (X86):
+    status: maintained
+    maintainers:
+        - jenmwms
+    collaborators:
+        - jhedberg
+        - aasthagr
+    files:
+        - boards/x86/
+        - soc/x86/
+    labels:
+        - "platform: X86"
+
+Intel Platforms (Xtensa):
+    status: maintained
+    maintainers:
+        - nashif
+    collaborators:
+        - andyross
+        - dcpleung
+    files:
+        - boards/xtensa/intel_*/
+        - soc/xtensa/intel_*/
+        - samples/boards/intel_s1000_crb/
+    labels:
+        - "platform: Intel CAVS"
+
+NXP Platforms:
+  status: maintained
+  maintainers:
+    - MaureenHelm
+  files:
+    - boards/arm/mimx*/
+    - boards/arm/frdm_k*/
+    - boards/arm/lpcxpress*/
+    - boards/arm/twr_*
+    - soc/arm/nxp_*/
+    - drivers/*/*imx*
+    - drivers/*/*lpc*
+    - drivers/*/*mcux*
+    - dts/arm/nxp/
+  labels:
+    - "platform: NXP"
+
+Microchip Platforms:
+  status: maintained
+  maintainers:
+    - scottwcpg
+  collaborators:
+    - VenkatKotakonda
+    - albertofloyd
+  files:
+    - boards/arm/mec*/
+    - dts/arm/microchip/
+    - soc/arm/microchip_mec/
+    - drivers/*/*mchp*
+  labels:
+    - "platform: Microchip"
+
+nRF Platforms:
+    status: maintained
+    maintainers:
+        - ioannisg
+    files:
+        - boards/arm/nrf*/
```

Comment:
I think `boards/arm/*nrf*/` would be better, to catch things like `thingy52_nrf52832`.